bundle/bind_test.go

```diff
@@ -17,43 +17,29 @@
 package bundle
 
 import (
-	"encoding/json"
 	"testing"
-
-	ft "github.com/stretchr/testify/assert"
 )
 
 func TestBind(t *testing.T) {
 	t.Skip("skipping bind until we can pass in a mock client")
-	output := []byte(`
-Login failed (401 Unauthorized)
-
-PLAY [all] *********************************************************************
+	/*output := []byte(`
```

Comment:
Any reason this is commented?

Reply:
because it was not doing anything. this test was skipped.